CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.4 (2021-07-26)
+This release adds major new features since the 2.3 release.
+
+**Features**
+
+**Experimental Features**
+
```

Comment:
TODO:
- [ ] Experimental features intro
- [ ] time_bucket_ng: seconds to years with CAGGs (seconds to weeks)
- [ ] adding additional nodes